### PR TITLE
Add verify bank account route

### PR DESCRIPTION
**Pull request by Neeraj-Ghodla**

### CODE REVIEW

File: src/fiat/bank-authorities.md [hunk 1]
```diff
@@ -191,6 +191,34 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities/WRhAxxWpTKb5U7p
 }
 ```
 
+## Verify a bank authority **EXPERIMENTAL**
```
Comment (ncjones) [inline src/fiat/bank-authorities.md:194]: bank account

File: src/fiat/bank-authorities.md [hunk 2]
```diff
@@ -163,10 +163,10 @@ required together.
 | :----- | :-------------------------------------------- | :----------------------------------------------------------------- |
 | 403    | {% break _ DIRECT_DEBIT_ALREADY_AUTHORIZED %} | This bank authority cannot be changed as all fields have been set. |
 
-## Verify a bank authority
+## Verify a bank authority **DEPRECATED**
```
Comment (ncjones) [inline src/fiat/bank-authorities.md:166]: deprecated features should point reader to alternative

File: src/fiat/bank-authorities.md [hunk 3]
```diff
@@ -191,6 +191,34 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities/WRhAxxWpTKb5U7p
...
+  -H "x-api-key: 1234" \
+  -H "content-type: application/json" \
+  -d '{ "verificationCode": "1111" }'
+```
+
+**Required Fields**
+
+|      Field       |  Type  |                                                                                                          Description                                                                                                                            |
+| :--------------- | :----- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| verificationCode | String | The code relating to the top up the user created as part of registering a bank account with Centrapay. This code will show up on their bank statement and will be used to verify that the user has access to the bank account being registered. |
```
Comment (ncjones) [inline src/fiat/bank-authorities.md:212]: long descriptions in tables should be pulled out into a separate paragraph. In this case this information is already covered in the paragraph above.

File: src/fiat/bank-authorities.md [hunk 4]
```diff
@@ -191,14 +202,44 @@ curl -X POST "https://service.centrapay.com/api/bank-authorities/WRhAxxWpTKb5U7p
...
-<span id="bank-authority-get"></span>
-## Get information about a bank account **EXPERIMENTAL**
+**Error Responses**
 
-{% endpoint GET https://service.centrapay.com/api/bank-accounts/${id} %}
+| Status |                           Code                            |                              Description                              |
+| :----- | :-------------------------------------------------------- | :-------------------------------------------------------------------- |
+| 403    | {% break _ BANK_ACCOUNT_ALREADY_VERIFIED %}               | The bank account is already verified                                  |
+| 403    | {% break _ BANK_ACCOUNT_VERIFICATION_ATTEMPTS_EXCEEDED %} | The number of times the user can verify the bank account has exceeded |
+| 403    | {% break _ BANK_ACCOUNT_MISMATCH %}                       | The top up and the bank account do not belong to the same account     |
```
Comment (Neeraj-Ghodla) [inline src/fiat/bank-authorities.md:211]: I am not sure what this error code should be called. Should it be BANK_AUTHORITY_ACCOUNT_MISMATCH or BANK_ACCOUNT_TOPUP_MISMATCH or BANK_ACCOUNT_MISMATCH?

**Review by Neeraj-Ghodla — COMMENTED** [review]
Made the description short.

File: src/fiat/bank-authorities.md (hunk 2 above)
Comment (Neeraj-Ghodla) [reply src/fiat/bank-authorities.md]: Link added

File: src/fiat/bank-authorities.md (hunk 1 above)
Comment (Neeraj-Ghodla) [reply src/fiat/bank-authorities.md]: Fixed

**Review by ncjones — APPROVED** [review]

**Pull request merged**